### PR TITLE
OSDOCS-3155 OSD/Rosa: Scaling/Autoscaling additions

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-3155

Preview builds: 
OSD: Managing worker nodes: https://deploy-preview-40901--osdocs.netlify.app/openshift-dedicated/latest/nodes/rosa-managing-worker-nodes.html
OSD About autoscaling nodes on a cluster: https://deploy-preview-40901--osdocs.netlify.app/openshift-dedicated/latest/nodes/nodes-about-autoscaling-nodes.html#cluster-autoscaler-about_nodes-about-autoscaling-nodes
ROSA About autoscaling nodes on a cluster: https://deploy-preview-40901--osdocs.netlify.app/openshift-rosa/latest/nodes/nodes-about-autoscaling-nodes.html#cluster-autoscaler-about_nodes-about-autoscaling-nodes

### CODE REVIEW

File: _topic_maps/_topic_map_osd.yml [hunk 1]
```diff
@@ -141,6 +141,8 @@ Distros: openshift-dedicated
 Topics:
 - Name: About machine pools
   File: nodes-machinepools-about
+- Name: Managing worker nodes
+  File: rosa-managing-worker-nodes
```
Comment (bmcelvee) [inline _topic_maps/_topic_map_osd.yml:145]: This is working on my local build, but not on netify yet.

Comment (bmcelvee) [reply _topic_maps/_topic_map_osd.yml:145]: And it appears that it's breaking the OSD netify on this PR altogether. I will try a different approach.

Comment (bmcelvee) [reply _topic_maps/_topic_map_osd.yml:145]: It builds locally and appears to be an issue with netify + the OSD topic map. I will link to a local build when ready to review.

**Comment by bmcelvee** [comment]
Thanks again for your help on this one, @okashi18! When you get a chance, would you mind giving it a review please?

**Comment by bmcelvee** [comment]
@xueli181114  would you mind giving this PR a QE review, please? Thanks!

**Comment by xueli181114** [comment]
@bmcelvee  Thanks for your notification. @yuwang-RH  Would you please review this DOC?

**Comment by yuwang-RH** [comment]
LGTM.

File: modules/cluster-autoscaler-about.adoc [hunk 2]
```diff
@@ -11,7 +11,14 @@ The cluster autoscaler adjusts the size of an {product-title} cluster to meet it
 
 The cluster autoscaler increases the size of the cluster when there are pods that fail to schedule on any of the current worker nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
 
-The cluster autoscaler computes the total memory, CPU, and GPU on all nodes the cluster, even though it does not manage the control plane nodes. These values are not single-machine oriented. They are an aggregation of all the resources in the entire cluster. For example, if you set the maximum memory resource limit, the cluster autoscaler includes all the nodes in the cluster when calculating the current memory usage. That calculation is then used to determine if the cluster autoscaler has the capacity to add more worker resources.
+The cluster autoscaler computes the total
```
Comment (bergerhoffer) [inline modules/cluster-autoscaler-about.adoc:14]: I'm not seeing this change in the OSD or ROSA preview. I tried following it down to see why it's not, but I gave up :) 

So I can't confirm for sure if this para renders okay or not w/ the ifdefs

File: modules/rosa-adding-node-labels.adoc [hunk 3]
```diff
@@ -18,6 +18,7 @@ Currently, adding node labels on an existing machine pool adds the labels to onl
 
 .Procedure
 
+ifdef::openshift-rosa[]
 . To create a new machine pool, add the node labels, and create replica worker nodes, enter the following command:
```
[the remark this answers is not in the document]
Comment (bmcelvee) [reply modules/rosa-adding-node-labels.adoc:22]: Thanks! It's an easy fix, so I'll go ahead and update it now.

**Comment by stevsmit** [comment]
LGTM

**Review by stevsmit — COMMENTED** [review]
LGTM!

**Review by stevsmit — APPROVED** [review]
LGTM!

**Pull request merged**